misk-hibernate/src/main/kotlin/misk/hibernate/actions/HibernateDatabaseQueryWebActionModule.kt

```diff
@@ -0,0 +1,55 @@
+package misk.hibernate.actions
+
+import com.google.inject.Injector
+import misk.exceptions.BadRequestException
+import misk.hibernate.Transacter
+import misk.inject.KAbstractModule
+import misk.inject.typeLiteral
+import misk.web.WebActionModule
+import misk.web.metadata.DatabaseQueryMetadata
+
+/** Install Hibernate specific Web Actions to support Database Query admin dashboard tab */
+class HibernateDatabaseQueryWebActionModule : KAbstractModule() {
+  override fun configure() {
+    install(WebActionModule.create<HibernateDatabaseQueryDynamicAction>())
+    install(WebActionModule.create<HibernateDatabaseQueryStaticAction>())
+  }
+
+  companion object {
+    /** Throw a readable error if a Dynamic Query is sent to the Static Web Action or vice versa */
+    fun checkDynamicQuery(queryClass: String, isDynamic: Boolean) {
+      val isDynamicQuery = queryClass.endsWith(DatabaseQueryMetadata.DYNAMIC_QUERY_KCLASS_SUFFIX)
+      val status = if (isDynamicQuery) "is" else "is not"
+      val action = if (isDynamic) {
+        HibernateDatabaseQueryStaticAction::class.simpleName!!
+      } else {
+        HibernateDatabaseQueryDynamicAction::class.simpleName!!
+      }
+
+      if ((isDynamic && !isDynamicQuery) || (!isDynamic && isDynamicQuery)) {
```

Comment:
`isDynamic != isDynamicQuery` ?

Reply:
`isDynamic` refers to if the callsite is the Dynamic Web Action or Static Web Action, renamed to reduce confusion